attractions/snackbar/snackbar.svelte

```diff
@@ -27,7 +27,10 @@
     <Button
       class={buttonClass}
       on:click={() => {
-        action.callback();
+        if (typeof action.callback === 'function') {
+          action.callback();
+        }
+
         if (closeOnAction) {
```

Comment:
Why isn't the check here similar to the previous one? It would help protect against bad input like `true`.
```suggestion
        if (typeof closeOnAction === 'function') {
```

Reply:
Um, because `true` is, in fact, the correct input here)

Reply:
Oh sh*t. I misread the following line to also be `closeOnAction` 😅